src/views/components/landing-top.html

```diff
@@ -217,6 +217,10 @@ <h2>fast, reliable, and automated</h2>
 				<div class="sponsor">
 					<a target="_blank" rel="noopener" href="https://bunny.net"><img src="{{@shared.assetsHost}}/img/landing/bunny.png" srcset="{{@shared.assetsHost}}/img/landing/bunny@2x.png 2x" title="Sponsored by bunny.net"></a>
 				</div>
+
+				<div class="sponsor">
+					<a target="_blank" rel="noopener" href="https://www.quantil.com"><img src="{{@shared.assetsHost}}/img/landing/quantil.png" srcset="{{@shared.assetsHost}}/img/landing/quantil@2x.png 2x" title="Sponsored by Quantil"></a>
```

Comment:
The png here needs to be resized to 50px in height (keep whitespace, just resize the whole thing).